src/controllers/ApiController.php

```diff
@@ -26,15 +24,8 @@ public function getItemId($request)
     public function getVerbosity($request)
     {
         // verbosity
-        if (isset($request->parameters['verbose'])
-            && $request->parameters['verbose'] == 'yes'
-        ) {
-            $verbose = true;
-        } else {
-            $verbose = false;
-        }
-
-        return $verbose;
+        return isset($request->parameters['verbose'])
```

Comment:
Wouldn't it be easier to read like this?

```php
if (! isset($request-parameters['verbose'])) {
    return false;
}

if ($request->parameters['verbose'] !== 'yes') {
    return false;
}

return true;
```

Reply:
I find my way cleaner and easier to read, but I'm not one of the core maintainers of the project ;)

5925a24